plugins/scaffolder-node/src/filters/createTemplateFilter.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CreatedTemplateFilter } from './types';
+
+/**
+ * This function is used to create new template filters in type-safe manner.
+ * @public
```

Comment:
This goes for really anything new exported that we're not super confident in yet.

Reply:
"Moving" the new stuff to alpha makes perfect sense, but by this do you mean actual physical organization or simply whether a given item is exported via `alpha` or `index`? In current state `filters` and `globals` are siblings to `actions`; I am restoring the preexisting types to their original location in `src/types.ts` but is it sufficient to have the new items exported through `alpha.ts` as well as being annotated `@alpha`?

Reply:
So they want to be exported from the `alpha` entry point. You can see the definition for which file that is by looking in the `package.json` under the `exports` property. 

I think you'll find a `alpha` or `next` folder in the respesctive packages too, and it's best if they're located under there instead of being intertwined with other code, makes it easier to understand what's going on and keeping them separate is a good thing.

Reply:
That's what I was saying; `plugins/scaffolder-node/src` has `alpha.ts` but no `alpha` subfolder. It sounds like you're saying the appropriate approach here would be to migrate `alpha.ts` to `alpha/index.ts` and move the new APIs under `alpha`.